bundle/compliance/cis_gcp/rules/cis_4_6/rule.rego

```diff
@@ -0,0 +1,23 @@
+package compliance.cis_gcp.rules.cis_4_6
+
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+
+default is_ip_forwarding_enabled = false
+
+# Ensure That IP Forwarding Is Not Enabled on Instances
+finding = result {
+	# filter
+	data_adapter.is_compute_instance
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(assert.is_false(is_ip_forwarding_enabled)),
+		{"Compute instance": input.resource},
+	)
+}
+
+is_ip_forwarding_enabled {
+	data_adapter.resource.data.canIpForward == true
+}
```

Comment:
would that be the same? if yes, we can move it into `finding`
```suggestion
is_ip_forwarding_enabled := data_adapter.resource.data.canIpForward
```

Reply:
I can modify it the way you suggested but it must reside outside of the finding clause - in case it will be undefined.